common/docsV1.md

```diff
@@ -249,6 +249,22 @@ Public keys verify that client token requests are coming from an authorized appl
 
 **1. Download the snippet.txt file located in the DPC Portal to create a signature.**
 
+- If you are unable to download the snippet.txt file with a GUI you can download the file using the command invocation below:
+
+  ~~~
+  curl -JLO https://raw.githubusercontent.com/CMSgov/dpc-app/master/dpc-web/public/snippet.txt 
+  ~~~
+
+**1a. Verify the type of file.**
+
+- If you are using a Mac or Linux shell you can check the file type with this command (will not work in a basic Windows environment)
```

Comment:
- Add a comma after "shell". 
- Replace "(will not work in a basic Windows environment)" with "This command will not work in a basic Windows environment.".
- Add period after "command" in the first sentence.

Updated sentences are: If you are using a Mac or Linux shell, you can check the file type with this command. This command will not work in a basic Windows environment.